share/spice/rand_word/rand_word.js

```diff
@@ -11,11 +11,19 @@
         while (api_result.length > 0) {
             list_of_list.push(api_result.splice(0, 4));
         }
-
+        
+        var title = 'Random Words';
+        var group = 'list';
+        
+        if ( api_result.length === 1 ) {
+            title = 'Random Word';
+            group ='text';
```

Comment:
I was wrong before, it'll be easier if you keep with the `list` group, but for the single would you need to specify `content: Spice.rand_word.single_word` (for example) Where `single_word.handlebars` is a new sub-template that create a link for a single word.

You'll need to play around with it to get it working. I think this will require you to get your own Wordnik API key